src/components/TeacherComponents/UnitListItemLessonCount/UnitListItemLessonCount.tsx

```diff
@@ -26,7 +26,6 @@ export const UnitListItemLessonCount = ({
           <OakSpan $font={["body-3", "heading-light-7"]} $color={textColor}>
             {!!lessonCount &&
               `${lessonCount} ${lessonCount > 1 ? "lessons" : "lesson"}`}
-            {expired && ` Coming soon`}
```

Comment:
perhaps changing the wording might be a better solution? to `Currently unavailable` or similar?

Reply:
For consistency, on other expired legacy lessons we say This lesson is currently unavailable.

Reply:
✅

Reply:
should probably say unit instead of lesson tho!

Reply:
Good spot, amended ✅